### PR TITLE
Update wordpres/components to fix VisuallyHidden styles not loading

### LINKED ISSUE #4015: Visually hidden text is not visually hidden
### To Reproduce

1. As a merchant, navigate to **Payments → Settings**.
2. Scroll down to the "Transactions and deposits" section.
3. Observe the text:
> Manage and update your deposit account information to receive payments and payouts. [Manage in Stripe(opens in a new tab)](https://alleged-okapi.jurassic.ninja/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments&wcpay-login=1&_wpnonce=1a2a83408e)

The "(opens in a new tab)" text should not be visible. It's present in the DOM for screen readers but shouldn't actually be visible. I noticed the span element uses the `components-visually-hidden` class but I do not see any CSS styles attached to that class.

### Actual behavior
Visually hidden text is not visually hidden.

### Screenshots
Screen Shot 2022-03-23 at 11 18 38 am

### Expected behavior
Visually hidden text is visually hidden.

### Desktop (please complete the following information):

* Device: MBP
* OS: macOS 11.2.3 (20D91)
* Browser: Google Chrome Version 98.0.4758.109 (Official Build) (x86_64)

### PR DESCRIPTION
**Pull request by oscarssanchez**
Fixes #4015

#### Changes proposed in this Pull Request

Update wordpres/components to fix VisuallyHidden styles not loading.

This PR updates the @wordpress/components dependency to `19.7.0`. It seems the old dependency has a bug where the `VisuallyHidden` component styles are not loaded by `ExternalLink`

#### Testing instructions

1.- Navigate to Payments → Settings.
2.- Scroll down to the "Transactions and deposits" section.
3.- See the text `opens in a new tab`  is now hiding correctly.

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) : _Add link here / 'QA Testing Not Applicable'_

### CODE REVIEW

**Comment by aprea** [comment]
Hi @oscarssanchez, thanks for this PR!

I noticed there are over 20,000 changes in the `package-lock.json` file.

Are we able to limit the changes in this file by explicitly updating the `@wordpress/components` package with:

```
npm install @wordpress/components@19.7.0 --save-exact
```

When running this locally I noticed the changeset is much smaller, i.e.

```
git diff --stat

 package-lock.json | 679 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++---------------
 package.json      |   2 +-
 2 files changed, 601 insertions(+), 80 deletions(-)
```

I can also see that the JS tests are failing, we might need to update the snapshots.

File: changelog/fix-4015-visually-hidden [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Update wordpres/components to fix VisuallyHidden styles not loading.
```
Comment (aprea) [inline changelog/fix-4015-visually-hidden:4]: **Nitpick:** Small typo here.

```suggestion
Update wordpress/components to fix VisuallyHidden styles not loading.
```

**Comment by shendy-a8c** [comment]
@oscarssanchez Thank you for this PR. As we're working on the settings page, we notice the issue that will be fixed by your PR. @aprea has suggested some feedbacks above and it looks to me that this PR is very close to fix the issue.

Since the last progress was made couple months ago, this PR's branch of your forked repo will need some syncing. Do you think you have time to make further progress in the near future? Otherwise, I'm happy to open a new PR based of your solution if it's OK with you.

**Comment by zmaglica** [comment]
Closing this pull request since the issue is not presented anymore. There is a big downside of upgrading dependencies. The tests which failed requires detail refactoring, since the update snapshot command didn't resolve the failed test. 

Please don't hesitate to create another issue if you consider that we need to update the components package, but since the issue is resolved by itself, I will close this PR since it requires more effort.

**Pull request closed without merging**